### DISCUSSION TITLE
Proposal: Merging tool to manually fix data

### DISCUSSION THREAD

**Discussion by GuillaumeSTEIN** [teslamate-org/teslamate#1725] [Ideas]
Hi,

I have noticed that some drives are split in 2 (without switching to Park).
![Capture d’écran 2021-06-15 à 00 40 46](https://user-images.githubusercontent.com/10887895/121968316-65a1cb00-cd72-11eb-9c7a-e34d03482b42.png)

Would it be possible to have a tool to merge 2 drives.
Maybe merging 2 charges might also be useful to someone else.

It might look like :

`docker-compose exec teslamate bin/teslamate rpc \
    "TeslaMate.Repo.get!(TeslaMate.Log.Drive, 864, 865) |> TeslaMate.Log.merge_drives()"`
    
Maybe ?
Thank you for reading me :)
This tool is gorgeous.

**Comment by adriankumpf** [comment]
Feature requests belong here: [💡 Ideas](https://github.com/adriankumpf/teslamate/discussions/categories/ideas). I'll move it.
